src/plugins/raven/sagas/connectionFlow.js

```diff
@@ -61,6 +64,23 @@ export default function* (raven) {
     attempts += 1;
     if (attempts > 10) {
       console.error('raven::connectionFlow::exceeded_10_attempts'); // eslint-disable-line no-console
+      const userWantsToReconnect = yield swal.fire({
+        title: 'Alert!',
+        text: 'Active Edits has disconnected, please Save and Refresh.',
+        type: 'info',
+        confirmButtonText: 'Thank You',
+        buttonsStyling: false,
+        customClass: {
+          confirmButton: 'btn-modal btn btn-outline-primary btn-block',
+        },
+      });
+
+      if (userWantsToReconnect.value && userWantsToReconnect.value === true) {
```

Comment:
@tbone1020 you can remove all this reconnect stuff since they dont want it anymore - it's just a notification now